definitions/virtual_machine.md

```diff
@@ -6,10 +6,11 @@ category: Technology
 ## Virtual Machine
 
 ### What it is
-A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. With this separation, processes can be executed using different operating systems on the same physical machine.
+A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. It will expose virtualised hardware resources (IRQ controllers, Timers, PCI devices) or will emulate them, depending on its capabilities and configuration.
+Virtual Machines allow for a complete operating system and the attached user space isolation from the physical machine. With this separation, processes can be executed using different operating systems and environments, while sharing the same hardware resources.
```

Comment:
Good start but unfortunately it takes us further from our goal. The idea for the glossary is to keep the language as accessible as possible for readers regardless of background. Terms like "user space isolation" aren't helpful in accomplishing that objective.